### ISSUE TITLE
Dark mode checkboxes and radio buttons don't follow the accent color

### ISSUE THREAD

**Issue by Valer100** [krlvm/AccentColorizer#51]
The dark mode checkboxes and radio buttons in some programs like Notepad++ don't follow the system's accent color when AccentColorizer is running.

![image](https://github.com/user-attachments/assets/08ed277b-c66d-43e7-a3fa-ab0ea920fdde) ![image](https://github.com/user-attachments/assets/fc6b3911-e5c8-4e77-9728-0c0943875cee)

AccentColorizer 1.2.0
Windows 11 23H2
Accent color: `#008B00`

**Comment by krlvm** [comment]
Is it a custom theme?

**Comment by Valer100** [comment]
No, it's the default Windows theme. It works with the normal checkbox and radio button, but it doesn't work with these dark mode styled ones that Notepad++ uses.

![image](https://github.com/user-attachments/assets/54d5ad6c-cb8e-4bac-99aa-4190b5d75733)

**Comment by WinExperiments** [comment]
Use my forked copy, it should work

**Comment by Valer100** [comment]
@WinExperiments ~I tried it, but it doesn't work. The dark mode checkboxes and radiobuttons are still blue when Notepad++'s theme is set to dark.~ Nevermind, it works

**Comment by rinyuuki7** [comment]
It also doesn't follow accent color when selecting folders in the windows explorer on dark mode

![Image](https://github.com/user-attachments/assets/57df6804-8b79-4000-9a97-fddc7a8925df)

**Comment by Undisputed00x** [comment]
> It also doesn't follow accent color when selecting folders in the windows explorer on dark mode
> 
> ![Image](https://github.com/user-attachments/assets/57df6804-8b79-4000-9a97-fddc7a8925df)

By default there isn't any accent color applied, your posted image also shows that.
